docker-compose.override.yml

```diff
@@ -2,13 +2,14 @@ version: '3'
 
 services:
   application:
-    image: larueli/php-symfony-base-image:8.0
+    container_name: application
     user: "1000:0"
     environment:
       DATABASE_URL: "mysql://etuutt:abcdef@database:3306/etuutt?serverVersion=mariadb-10.6.3&charset=utf8"
       MAILER_DSN: "smtp://mailer:1025"
     volumes:
       - ./:/var/www/html
+      - /var/www/vendor/

```

Comment:
Où est monté ce volume ? Normalement le `./:/var/www/html` monte tout le contenu (y compris le dossier vendor)

Reply:
Sans cette ligne, quand on copie le contenu de `./` dans `/var/www/html`, ceci vient écraser les vendor qui étaient installées par ton image `larueli/php-symfony-base-image`.
Je te laisse lire [ce topic](https://forums.docker.com/t/running-composer-install-when-container-is-created/119777/4) pour comprendre plus en profondeur. [Cette question stackoverflow](https://stackoverflow.com/questions/29181032/add-a-volume-to-docker-but-exclude-a-sub-folder) reprend le même problème pour le `node_modules`